search-app/src/cmr/search/services/acl_service.clj

```diff
@@ -36,10 +37,16 @@
   true)
 
 ;; generic concepts currently have no catalog item ACLs, so return `true` for all ACL checks
+;; except for draft concepts which checks provider object ACL PROVIDER_CONTEXT.
 (doseq [concept-type (cc/get-generic-concept-types-array)]
-  (defmethod acls-match-concept? concept-type
-    [context acls concept]
-    true))
+  (if (string/includes? (name concept-type) "draft")
```

Comment:
Theres an` is-draft-concept?` function in `common-lib/src/cmr/common/concepts.clj` we could use here

Reply:
if you do use those we need to import [cmr.common.concepts :as concepts]

Reply:
Done.